python/task/corpora.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Corpus locations"""
+
+import os
+import urllib2
+import sling.flags as flags
+import sling.log as log
+
+# Command-line flags.
+flags.define("--language",
+             help="primary language for resources",
+             default="en",
+             metavar="LANG")
+
+flags.define("--languages",
+             help="list of languages to process",
+             metavar="LANG,...")
+
+flags.define("--wikidata",
+             help="wikidata version",
+             #default="20161031",
+             default="20180101",
+             metavar="YYYYMMDD")
+
+flags.define("--wikipedia",
+             help="wikipedia version",
+             #default="20161101",
+             default="20180101",
+             metavar="YYYYMMDD")
+
+def post_process_flags(arg):
+  if arg.languages == None:
+    arg.languages = arg.language
+  if arg.languages == "ALL":
+    arg.languages = "en,da,sv,no,de,fr,es,it,nl,pt,pl,fi"
+  arg.languages = arg.languages.split(",")
+
+flags.hook(post_process_flags)
+
+def wikidata_url():
+  """URL for downloading Wikidata dump."""
+  return "https://dumps.wikimedia.org/wikidatawiki/entities/" + \
```

Comment:
I think you can probably set the "wiki date" to be latest, but I will not encourage using the latest versions since this makes version tracking faster. The idea is that we update the wikipedia and wikidata version flags to the current version when we have tested that this version works.